cmd/clusterctl/examples/ibmcloud/machines.yaml.template

```diff
@@ -7,50 +7,51 @@ items:
     generateName: ibmcloud-master-
     labels:
       set: master
-      cluster.k8s.io/cluster-name: "test1"
+      cluster.k8s.io/cluster-name: "CLUSTER_NAME"
   spec:
     providerSpec:
       value:
         apiVersion: "ibmcloudproviderconfig/v1alpha1"
         kind: "IBMCloudMachineProviderSpec"
-        domain: <domain name>
-        maxMemory: 4096
-        startCpus: 4
-        dataCenter: <datacenter name>
-        osReferenceCode: <the operating system selection>
+        domain: CONTROL_PLANE_DOMAIN
```

Comment:
what about `s/CONTROL_PLANE/MACHINE/`? control plane makes less sense for machines I guess.